### PR TITLE
Awesomebooks with modules

### PR DESCRIPTION
**Pull request by AnsarIbrahim**
### This PR Contains
- Create a new directory and initialize it with git ✅
- Use the npm init -y command to create a package.json file ✅
- Create the entry point for your JavaScript code called index.js  ✅
- Create an index.css file for your styles ✅
- Create the main index.html file. Link your index.js and index.css files there ✅
- Create a directory called modules ✅
- Use ES6 syntax ✅
- Divide your code into modules ✅
- Create a separate file for each module in the modules directory ✅
- Import modules into the index.js file ✅
- Keep all functionalities of the app without errors ✅
- Refactor your functions to arrow functions ✅
- For this requirement, you should use a library [Luxon] ✅
- Download the library into your project using npm ✅
- Import it as a module ✅
- Set current date and time ✅

### CODE REVIEW

File: modules/display_books.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    const author = this.authorInput.value;
+    if (title && author) {
+      const newBook = { title, author };
+      this.books.push(newBook);
+      localStorage.setItem('books', JSON.stringify(this.books));
+      this.titleInput.value = '';
+      this.authorInput.value = '';
+      this.bookElement(newBook, this.books.length - 1);
+    }
+  }
```
Comment (binhussen) [inline modules/display_books.js]: - Great job for implementing using class, I think in this milestone you have to use an arrow function to fit the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) 👍 
Screenshot 2023-06-05 154539

File: modules/display_books.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  bookElement(book, index) {
+    const bookContainer = document.createElement('td');
+    if (index % 2 === 0) {
+      bookContainer.classList.add('dark');
+    }
```
Comment (binhussen) [inline modules/display_books.js]: - Great job for implementing using class, I think in this milestone you have to use an arrow function to fit the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) 👍 
Screenshot 2023-06-05 154539

File: modules/display_books.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    this.booksParent.appendChild(bookContainer);
+    bookContainer.scrollIntoView({ behavior: 'smooth' });
+  }
+
+  displayBooks() {
+    this.booksParent.innerHTML = '';
+    this.books.forEach((book, index) => {
+      this.bookElement(book, index);
+    });
+  }
```
Comment (binhussen) [inline modules/display_books.js]: - Great job for implementing using class, I think in this milestone you have to use an arrow function to fit the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) 👍 
Screenshot 2023-06-05 154539

File: modules/display_books.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+  deleteBook(index) {
+    this.books.splice(index, 1);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.displayBooks();
+  }
+}
```
Comment (binhussen) [inline modules/display_books.js]: - Great job for implementing using class, I think in this milestone you have to use an arrow function to fit the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) 👍 
Screenshot 2023-06-05 154539

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @AnsarIbrahim , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Well Stractured Code✔️
- Sticked Given WIreframe ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AnsarIbrahim, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**